client/app/bundles/shared/components/Footer/Connect.jsx

```diff
@@ -1,25 +1,47 @@
+// @flow
 import React from 'react';
-import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import { defaultMessages } from 'libs/i18n/default';
 import css from './Footer.scss';
 
-const Connect = props => (
-  <ul>
-    <h6 className={css.footer_header}>Connect</h6>
-    <li><a href="mailto:join.ifme@gmail.com" target="blank">{props.common.form.email}</a></li>
-    <li><a href="http://facebook.com/ifmeorg" target="blank">{props.navigation.facebook}</a></li>
-    <li><a href="https://github.com/ifmeorg/ifme" target="blank">{props.navigation.github}</a></li>
-    <li><a href="https://www.instagram.com/ifmeorg" target="blank">{props.navigation.instagram}</a></li>
-    <li><a href="https://medium.com/ifme" target="blank">{props.navigation.medium}</a></li>
-    <li><a href="https://opencollective.com/ifme" target="blank">{props.navigation.opencollective}</a></li>
-    <li><a href="http://patreon.com/ifme" target="blank">{props.navigation.patreon}</a></li>
-    <li><a href="https://medium.com/feed/ifme" target="blank">{props.navigation.rss}</a></li>
-    <li><a href="http://twitter.com/ifmeorg" target="blank">{props.navigation.twitter}</a></li>
-  </ul>
-);
-
-Connect.propTypes = {
-  navigation: PropTypes.string.isRequired,
-  common: PropTypes.string.isRequired,
+type Prop = {
+  intl: Object
 };
 
+const Connect = injectIntl(({ intl }: Prop) => {
+  const { formatMessage } = intl;
+  return (
+    <ul>
+      <h6 className={css.footer_header}>Connect</h6>
+      <li><a href="mailto:join.ifme@gmail.com" target="blank">
```

Comment:
Hmmm... shouldn't it be target="_blank"? If we intend to route all clicks to the same tab/window, then this is fine, but there should be a comment in the code indicating that intent (kinda like how people use "[sic]" to quote intentional misspellings).